packages/react-query/package.json

```diff
@@ -70,7 +70,7 @@
     "tsd": "^0.28.1"
   },
   "peerDependencies": {
-    "@suspensive/react": "^1.11.4-beta.0",
+    "@suspensive/react": "workspace:1.14.8",
```

Comment:
Can users can only use this version?!

Reply:
I dont know well in this part
Isn't it common to use ^or an upper(>=) version?!

Reply:
Yes, I wanted to force users to use fixed versions of all suspensive packages
I thought that if we provided fixed versions of all suspensive packages as same package version, it would be easier for not only our maintainers and also for our users to manage versions as well.